purchase_fop_shipping/README.rst

```diff
@@ -0,0 +1,58 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License AGPL-3
+
+===========================
+Pruchase Free-Of-Paiment shipping
+===========================
+
+This module allows to define a Free-Of-Paiment (FOP) shipping on supplier.
+FOP shipping is a min order amount to got shipping free from supplier.
+User can force confirm order if he/she wishes. 
+
+Installation
+============
+
+You need an Odoo/OCB installation v 10.0.
```

Comment:
i guess that such lines are not necessary. You can freely remove all the empty (usage) or default section (installation)